mock/mockimpl/kvproc/basic.go

```diff
@@ -57,9 +57,13 @@ func (e *Engine) parseExpiry(expiry uint32) time.Time {
 
 	// TODO(brett19): Check if this is the right edge for expiry.
 	if expiry > 30*24*60*60 {
-		return time.Unix(int64(expiry), 0)
+		return time.Unix(int64(expiry), 0).Add(e.db.Chrono().TimeShift())
 	}
 
 	expiryDura := time.Duration(expiry) * time.Second
 	return e.db.Chrono().Now().Add(expiryDura)
 }
+
+func (e *Engine) HLC() time.Time {
```

Comment:
This needs to return the same format as the CAS code above, since I believe the $vbucket.HLC thing returns both the hybrid AND the clock portions.

Reply:
The HLC just returns the current timestamp from the clock in seconds, from what I can tell.